readthedocs/templates/builds/build_detail.html

```diff
@@ -161,6 +162,18 @@ <h3>{% trans "Error" %}</h3>
            data-bind="text: error">
           {{ build.error }}
         </p>
+        <p>
+        {% blocktrans with build_id=build.pk proj_name=build.project.name build_path=request.get_full_path user_name=request.user %}
+         Report any build issues <a href="https://github.com/rtfd/readthedocs.org/issues/new?
+         title=Build%20error%20with%20build%20id%20%23{{ build_id }}
+         &body=%23%23%20Details%0A%0A
+         *%20Project%20Url%3A%20https://readthedocs.org/projects/{{ proj_name }}/%0A
+         *%20Build%20URL%20(if%20applicable)%3A%20https://readthedocs.org{{ build_path }}%0A
```

Comment:
`build_path` already includes the `/`, right?

`Project Url`, `URL` uppercase (same as the template) 

Also there is some way of django to do the encoding here? Something like https://github.com/rtfd/readthedocs.org/pull/3457/files#diff-02c668683a9a481865c186f12f7452e0R74

Reply:
Yup `build_path` includes `/`.